css/badge.css

```diff
@@ -0,0 +1,118 @@
+.badge {
+  padding: 0.2rem 0.4rem;
+  font-weight: 600;
+  border-radius: 5px;
+  letter-spacing: 2px;
+  color: var(--rb-very-light-grey);
+}
+
+.badge.primary {
+  background-color: var(--rb-primary);
+}
+
+.badge.secondary {
+  background-color: var(--rb-dark-grey);
+  color: var(--rb-very-light-grey);
+}
+
+.badge.success {
+  background-color: var(--rb-green);
+  color: var(--rb-very-light-grey);
+}
+
+.badge.warning {
+  background-color: var(--rb-yellow);
+  color: var(--rb-light-black);
+}
+
+.badge-icon {
+  position: relative;
+  cursor: pointer;
+}
+
+.badge-icon i:hover {
+  color: var(--rb-dark-grey);
+}
+
+.badge-i {
+  position: absolute;
+  top: 0;
+  right: -0.5rem;
+  font-size: 1.1rem;
+  padding: 2px 0.6rem;
+  background-color: var(--rb-primary);
+  color: var(--rb-very-light-grey);
+  border-radius: 50%;
+}
+
+.indicator {
+  position: absolute;
+  z-index: 2;
+  border-radius: 50%;
+  background-color: var(--rb-green);
+  border: 2px solid var(--rb-very-light-grey);
+}
+
+.ind-xs {
+  bottom: 0.5rem;
+  right: 0;
+  width: 5px;
+  height: 5px;
+}
+.ind-s {
+  bottom: 0.5rem;
+  right: 0.1rem;
+  width: 10px;
```

Comment:
Good practice to avoid using multiple units. Here both rem and px are being used so you can use rem for width and height as well.